README.md

```diff
@@ -1 +1,5 @@
-# whatsfordinner
\ No newline at end of file
+# whatsfordinner
+Story
+Tammy wasn’t much of a cook, she knew her husband loved to make new and exciting meals and drinks. 
+For their anniversary she decided even if she didn’t know her way around the kitchen she could still surprise her husband by having all the ingredients needed for whatever he wanted to make. 
+She commissioned an app to be made so whatever protein or alcohol they had in the house could be searched and a list of meals, cocktails, and most importantly ingredients she could save and get to prepare for the meal.
```

Comment:
I would change the ingredients here to recipes.  Since we are saving the recipes and the user is entering in the ingredient.